Automattic/jetpack
# Boost: Add cornerstone pages support

### TODO:

- [x] https://github.com/Automattic/jetpack/issues/39945
- [x] https://github.com/Automattic/jetpack/issues/39948
- [x] https://github.com/Automattic/jetpack/issues/39949
- [x] https://github.com/Automattic/boost-cloud/issues/632
- [x] https://github.com/Automattic/jetpack/issues/39967

## Proposed changes:

* Add cornerstone pages feature.

### Other information:


## Jetpack product discussion

n/a

## Does this pull request change what data or activity we track or use?

no

## Testing instructions:

- Setup Boost;
- Test the cornerstone pages UI to make sure it works as expected.
